README.md

```diff
@@ -1,322 +1,329 @@
 
-This driver is on NPM.
+This driver can only be used with FQL X, and is not compatible with earlier versions of FQL. To query your databases with earlier API versions, see the [faunadb](https://www.npmjs.com/package/faunadb) package.
 
-You can install with npm with `npm install fauna`.
-You can install with yarn with `yarn add fauna`
+<details>
+<summary>Table of Contents</summary>
 
-## Pre-requisites
+- [A JavaScript driver for Fauna.](#a-javascript-driver-for-fauna)
+- [Compatibility](#compatibility)
+- [Installation](#installation)
+  - [Node.js](#nodejs)
+  - [Browsers](#browsers)
+- [Usage](#usage)
+  - [Connecting from the browser](#connecting-from-the-browser)
+  - [Importing into a bundled project](#importing-into-a-bundled-project)
+  - [Quick-Start](#quick-start)
+  - [Querying with FQL X](#querying-with-fql-x)
+  - [Query Composition](#query-composition)
+  - [Typescript Support](#typescript-support)
+  - [Query Options](#query-options)
+  - [Client Configuration](#client-configuration)
+    - [Using environment variables](#using-environment-variables)
+  - [Query Statistics](#query-statistics)
+- [Contributing](#contributing)
+  - [Setting up this Repo](#setting-up-this-repo)
+  - [Running tests](#running-tests)
+  - [Linting your code](#linting-your-code)
+- [License](#license)
 
-If you have not already created a database and access key (secret) in the Preview environment please do so now. Contact product@fauna.com for access to Preview.
+</details>
 
-## Creating a connection to your database
+# Compatibility
 
-The following code example sets up a connection to Fauna's Preview environment, authenticates with the secret you provided, and executes a simple query.
+This Driver supports and is tested on:
 
-```typescript
-import { Client, endpoints, fql } from "fauna";
-
-const client = new Client({
-  endpoint: endpoints.preview,
-  secret: "<my_fauna_secret>",
-});
-```
+- Node.js [Current and Active LTS](https://nodejs.org/en/about/releases/)
+  - Current - v19
+  - Active LTS - v18
+- Chrome
+- Firefox
+- Safari
+- Edge
 
-Default settings are available for the client. So you can creat a client as simply as:
+# Installation
 
-```typescript
-import { Client } from "fauna";
+## Node.js
 
-const client = new Client();
-// secret defaults to whatever is stored in a FAUNA_SECRET environmental variable
-// query_timeout_ms defaults to letting Fauna's service control the query timeout (maintained to meet typical customer workloads).
-// endpoint defaults to endpoints.cloud
+```shell
+npm install fauna
 ```
 
-The following snippet uses the client object, just created in the previous example, to execute queries in Fauna.
+or
 
-```typescript
-(async () => {
-  try {
-    var response = await client.query(fql`"Hello World!"`);
-    console.log(response);
-  } catch (e) {
-    console.log(e);
-  }
-})();
+```shell
+yarn add fauna
 ```
 
-## Export your secret as an environment variable
+## Browsers
```

Comment:
This feels like an edge case.

Calling it "browsers" feels a little odd as if i'm writing code to run in a browser i'm probably going to use an actual package manager and not whack the driver straight into my script.

```suggestion
## Available via CDN
```